addons/flannel/template/base/install.sh

```diff
@@ -274,6 +280,28 @@ function remove_weave() {
     rm -rf /var/lib/weave
     rm -rf /etc/cni/net.d/*weave*
     rm -rf /opt/cni/bin/weave*
+
+    logStep "Verifying Weave removal"
+
+    for resource in "${resources[@]}"; do
+        if kubectl -n kube-system get "$resource" &> /dev/null; then
+            logWarn "Resource: $resource still exists. Attempting force deletion."
+            if ! spinner_until 30 kubectl -n kube-system delete "$resource" --force --grace-period=0 1>/dev/null; then
```

Comment:
You cannot use a spinner function for a long running command as mentioned by @laverya